04-onion-routing.md

```diff
@@ -367,6 +381,239 @@ otherwise meets the amount criterion (eg. some other failure, or
 invoice timeout), however if it were to fulfill only some of them,
 intermediary nodes could simply claim the remaining ones.
 
+### Route Blinding
+
+Nodes receiving onion packets may hide their identity from senders by
+"blinding" an arbitrary amount of hops at the end of an onion path.
+
+When using route blinding, nodes find a route to themselves from a given
+"introduction node". They then use ECDH with each node in that route to create
+a "blinded" node ID and an encrypted blob (`encrypted_data`) for each one of
```

Comment:
See comments below for more context but I don't understand the extra use coming from the additional complexity of the blinded node id. Is that just a name for the variable to explain the formulas?

The actual encrypted node_ids need to be stored in `next_node_id` anyway and the blinded id can always be computed from the ephemeral shared secret and the actual node id (as currently specified in line 473 to 475) but not the other way around. So the blinded node ids seem to be useless to me.